meta/main.yml

```diff
@@ -7,12 +7,12 @@ galaxy_info:
   license: BSD
   min_ansible_version: 2.4
   platforms:
-    - name: EL
+    - name: Rocky Linux
```

Comment:
This does not seem to be defined as an official platform https://galaxy.ansible.com/api/v1/platforms/?name__icontains=rocky.
Proposing to keep EL as the overarching platform covering RHEL & Rocky Linux

Reply:
thanks, I did not know where to look for that. What should the version be then? Is `9` common to all Red had derived distros?